pkg/opencontainer/containers.go

```diff
@@ -150,6 +155,7 @@ func (l *Launchable) Executables(serviceBuilder *runit.ServiceBuilder) ([]launch
 			p2exec.P2ExecArgs{
 				NoLimits: true,
 				WorkDir:  l.InstallDir(),
+				EnvDirs:  []string{l.EnvDir()}, // TODO: add pod env dir
```

Comment:
I'm not at all sure that `runc`'s environment gets propagated inside the container.  Probably the correct way to approach this is to write the environment variables to the container's [`config.json`](https://github.com/opencontainers/specs/blob/master/config.md#process-configuration).

I'd prefer that you leave this out for the time being.